youtube_dl/extractor/detik.py

```diff
@@ -0,0 +1,67 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import ExtractorError
+
+
+class DuaPuluhDetikIE(InfoExtractor):
```

Comment:
Instead of this extractor you should add an extractor for `https://20.detik.com/embed/...` URLs and add detection of such embeds in generic extractor.

Reply:
Alright. I decided to just remove the extractor and instead add the detection and extractor into generic. Please check my latest commit. What do you think? Is it the correct way to do it in generic extractor?